projects/packages/forms/src/blocks/form-progress-indicator/edit.js

```diff
@@ -1,31 +1,75 @@
 import { useBlockProps } from '@wordpress/block-editor';
+import { SVG, Path } from '@wordpress/components';
 import StepControls from '../shared/components/form-step-controls';
 import useParentFormClientId from '../shared/hooks/use-parent-form-client-id';
 import useStepNavigation from '../shared/hooks/use-step-navigation';
+import { calculateProgressPercentage } from '../shared/util/progress-calculation';
 
 import './editor.scss';
 
 const FormProgressIndicatorEdit = ( { clientId } ) => {
 	const parentFormId = useParentFormClientId( clientId );
 	const { currentStepInfo, steps } = useStepNavigation( parentFormId );
 
-	let progress = steps.length ? ( ( currentStepInfo.index + 1 ) / steps.length ) * 100 : 10;
-	if ( currentStepInfo.index === -1 && steps.length > 0 ) {
-		progress = ( 1 / steps.length ) * 100; // Assume the first step is active
-	}
-
 	const blockProps = useBlockProps();
+	const isDotStyle = blockProps.className && blockProps.className.includes( 'is-style-dots' );
```

Comment:
```suggestion
	const isDotStyle = blockProps.className?.includes( 'is-style-dots' );
```

This would work too if `className` doesn't exist, it'll return `undefined`.